docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/ConvertTypeMultiColumn.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    // This example illustrates how to convert multiple columns of different types to one type, in this case System.Single. 
+    // This is often a useful data transformation before concatenting the features together and passing them to a particular estimator.
+    public static class ConvertTypeMultiColumn
+    {
+        // The initial data type
+        private class InputData
+        {
+            public bool Feature1;
+            public string Feature2;
+            public DateTime Feature3;
+            public double Feature4;
+        }
+
+        // The resulting data type after the transformation
+        private sealed class TransformedData : InputData
```

Comment:
>sealed [](start = 16, length = 6)

we don't need to worry about sealed. it might give the wrong impression that the data classes should be sealed.

I added this note to the checklist